pytorch_lightning/trainer/trainer.py

```diff
@@ -1011,7 +1011,7 @@ def run_pretrain_routine(self, model: LightningModule):
 
             eval_results = self._evaluate(model,
                                           self.val_dataloaders,
-                                          self.num_sanity_val_steps,
+                                          [self.num_sanity_val_steps],
```

Comment:
```suggestion
                                          [self.num_sanity_val_steps] * len(self.val_dataloaders)
```
maybe use keywoard args here to make it clear what each argument is